src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/enums/DependantChild.java

```diff
@@ -11,7 +14,8 @@
 //                "count": 2
 //            }
 //     ]
-
+@Setter
+@Getter
```

Comment:
would this not be better with an AllArgsConstructor instead? Think this should be generated from the schema as it is in CMA2